pycontrails/datalib/gfs/gfs.py

```diff
@@ -519,23 +531,24 @@ def _open_gfs_dataset(self, filepath: str | pathlib.Path, t: datetime) -> xr.Dat
             else:
                 v = variable
 
-            tmpds = xr.open_dataset(
-                filepath,
-                filter_by_keys={"typeOfLevel": v.level_type, "shortName": v.short_name},
-                engine="cfgrib",
-            )
-
-            if ds is None:
-                ds = tmpds
-            else:
-                ds[v.short_name] = tmpds[v.short_name]
+            try:
+                da = xr.open_dataarray(
+                    filepath,
+                    filter_by_keys={"typeOfLevel": v.level_type, "shortName": v.short_name},
+                    engine="cfgrib",
+                )
+            except ValueError as exc:
+                # To debug this situation, you can use:
+                # import cfgrib
+                # cfgrib.open_datasets(filepath)
+                msg = f"Variable {v.short_name} not found in {filepath}"
+                raise ValueError(msg) from exc
 
-            # set all radiation data to np.nan in the 0th step
             if is_radiation_step_zero:
-                ds = ds.rename({Visibility.short_name: variable.short_name})
-                ds[variable.short_name] = np.nan
+                da = xr.full_like(da, np.nan)  # set all radiation data to np.nan in the 0th step
+            da_dict[variable.short_name] = da
 
-        assert ds is not None, "No variables were loaded from grib file"
+        ds = xr.Dataset(da_dict)
```

Comment:
FYI -- this is all cosmetic and provides a better traceback for the issue found in #253